README.md

```diff
@@ -192,17 +190,10 @@ pop call contract -p ./my_contract --contract $INSTANTIATED_CONTRACT_ADDRESS --m
 ## E2E testing
 
 For end-to-end testing you will need to have a Substrate node with `pallet contracts`.
-You do not need to run it in the background since the node is started for each test independently.
-To install the latest version:
+Pop provides the latest version out-of-the-box by running:
 
-```
-cargo install contracts-node --git https://github.com/paritytech/substrate-contracts-node.git
-```
-
-If you want to run any other node with `pallet-contracts` you need to change `CONTRACTS_NODE` environment variable:
-
-```
-export CONTRACTS_NODE="YOUR_CONTRACTS_NODE_PATH"
```

Comment:
This is used to point to the `contracts-node` binary which is needed for running the e2e tests.

`e2e-tests` use the `contracts-node` binary to test against.

If you want to run e2e tests against a live contracts node you can look here:
- https://github.com/use-ink/ink-docs/pull/283/files#diff-b6664aae173c30e529697e60b09951bdcad954788b1452c97fc352514065e88cR267

Reply:
Yes... but if you're providing now a node inside the tool, why would a user want to do that?

Reply:
So how do you run the e2e tests?

Reply:
I just restored that part.